db/migrate/20230210104310_add_device_data_to_braintree_sources.rb

```diff
@@ -0,0 +1,5 @@
+class AddDeviceDataToBraintreeSources < ActiveRecord::Migration[5.1]
+  def change
+    add_column :solidus_paypal_braintree_sources, :device_data, :string
```

Comment:
I'm okay with adding `device_data` to the `braintree_sources` table since it's useful for debugging.

Reply:
Good to have it, and the table is fully owned by the extension, so no concerns on that side 👍

Reply:
I think it's ok :+1: